libvirt/egress.xml

```diff
@@ -8,7 +8,7 @@
   <bridge name='m3-egress' stp='on' delay='0'/>
   <ip address='192.168.125.1' netmask='255.255.255.0'>
     <dhcp>
-      <range start='192.168.125.2' end='192.168.125.254'/>
+      <range start='192.168.125.100' end='192.168.125.254'/>
```

Comment:
I missed that we're using `.100` for the network-infra host static IP:

```
metal3_network_infra_public_ip: 192.168.125.100
```

So we either need to change that IP to <100 or make this range start >100 (perhaps just make this start at 110?)